ext_mongodb.php

```diff
@@ -1,6 +1,12 @@
 <?hh
 namespace MongoDB\Driver;
 
+interface TypeWrapper
+{
+	function fromType(\MongoDB\BSON\Type $type);
+	function toType() : \MongoDB\BSON\Type;
```

Comment:
`use MongoDB\BSON\Type;` at the top of the file would let you abbreviate the interface as simply `Type` throughout the rest of the file.